packages/app/src/components/transactions/infoTable/GeneralInfo.vue

```diff
@@ -40,7 +40,16 @@
           </InfoTooltip>
         </TableBodyColumn>
         <TableBodyColumn class="transaction-table-value transaction-reason-value">
-          {{ transaction.error || transaction.revertReason || "" }}
+          <div v-if="transaction?.error && transaction?.revertReason">
```

Comment:
This is not right, means it is only shown if both error and revertReason are present which is not always the case